jsonschema-validation.xml

```diff
@@ -552,7 +552,7 @@
                     an array, elements of the array MUST be strings and MUST be unique.
                 </t>
                 <t>
-                    String values MUST be one of the seven primitive types defined by
+                    String values MUST be one of the six primitive types defined by
```

Comment:
Should the

Recall:  "number" includes "integer".

bit on line 560 also be removed?  Since there is no longer an "integer" primitive type, that sentence is a bit confusing.